src/Form/FieldInput.js

```diff
@@ -26,27 +26,32 @@ export default class FieldInput extends React.Component {
   }
 
   handleEvent(event, eventObj) {
-    this.props.handleEvent(
-      event, this.props.name, eventObj.target.value, eventObj
-    );
+    let {props} = this;
+    props.handleEvent(event, props.name, eventObj.target.value, eventObj);
```

Comment:
I like `let {props} = this;`
but when we only make use of `props` once, it just seems silly to assign a variable to a variable without modifying to only use it once.